fdbserver/ptxn/test/Driver.actor.cpp

```diff
@@ -214,6 +214,229 @@ void startFakeStorageServer(std::vector<Future<Void>>& actors, std::shared_ptr<T
 	}
 }
 
+namespace details {
+
+void TLogGroupFixture::setUp(TestDriverContext& testDriverContext,
+                             const int numTLogGroups,
+                             const int numStorageTeamIDs) {
+
+	ASSERT(numTLogGroups <= numStorageTeamIDs);
+
+	test::print::PrintTiming printTiming("TLogGroupFixture::setUp");
+
+	for (int i = 0; i < numTLogGroups; ++i) {
+		tLogGroupIDs.push_back(randomUID());
+	}
+	storageTeamIDs = generateRandomStorageTeamIDs(numStorageTeamIDs);
+
+	// Assign storageTeamIDs to TLog groups
+	for (const auto& tLogGroupID : tLogGroupIDs) {
+		tLogGroupStorageTeamMapping[tLogGroupID];
```

Comment:
is this right?

Reply:
This is expected, to touch the map so tLogGroupID is created in the mapping.